src/operator/tensor/elemwise_unary_op_basic.cu

```diff
@@ -77,6 +77,9 @@ NNVM_REGISTER_OP(_identity_with_attr_like_rhs)
 NNVM_REGISTER_OP(reshape_like)
 .set_attr<FCompute>("FCompute<gpu>", UnaryOp::IdentityCompute<gpu>);
 
+NNVM_REGISTER_OP(shape)
```

Comment:
Where does the name come from? This looks confusing and conflicts with the property name `shape` of `NDArray` in Python. Need to ensure the documentation can be rendered correctly.

Reply:
what do you suggest the name of the operator should be? This is more or less the name used in a couple of other frameworks too.

Reply:
Please make sure the doc page can be rendered correctly at least.